filter_plugins/list-dict-diff.py

```diff
@@ -8,9 +8,62 @@ def get_remaining_items(list_a, list_b, key_a, key_b):
 
     return list_a
 
+# For manage-workflow-templates role
```

Comment:
Let's either consider creating a new `.py` file, or otherwise rename this one as this new code is a bit "hidden" inside of this `list-dict-diff.py` file as-is.